lightning-invoice/src/utils.rs

```diff
@@ -21,6 +21,107 @@ use core::convert::TryInto;
 use core::ops::Deref;
 use core::time::Duration;
 
+#[cfg(feature = "std")]
+/// Utility to create an invoice that can be paid to one of multiple nodes, or a "phantom invoice."
+/// This works because the invoice officially pays to a fake node (the "phantom"). Useful for
+/// load-balancing payments between multiple nodes. Note that there are no privacy advantages to
```

Comment:
I'd still call it load-balancing, just because its not guaranteed doesn't mean we don't call other schemes load-balancing. https://github.com/lightningdevkit/rust-lightning/pull/1199#discussion_r803836981